src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -163,12 +163,11 @@ class CoreWorkerDirectTaskSubmitter {
   /// if a remote raylet tells us to spill the task back to the local raylet.
   const ClientID local_raylet_id_;
 
-  /// A function to override actor creation. The callback will be called once the actor
-  /// creation task has been accepted for submission, but the actor may not be created
-  /// yet.
-  std::function<Status(const TaskSpecification &task_spec,
-                       const gcs::StatusCallback &callback)>
-      actor_create_callback_;
+  /// An interface to create actor. It provides two methods, one of them is
+  /// `RegisterActor` which is synchronous, while another is
+  /// `AsyncCreateActor` which is invoked when the dependencies of the
+  /// actor are all resolved.
```

Comment:
I think it's okay to just document something like `Interface for actor creation` here. The details of this interface should be documented on the interface itself.